### PR TITLE
Fix: provider used does not work on 14.04

### PR DESCRIPTION
**Pull request by till**
# Changes

Provider on 14.04 is not `Init`, we fall back to the Chef default (which worked before).

### CODE REVIEW

**Comment by till** [comment]
@fh @gilleyj I tested this with a `pdf-autocite` vagrant box (uses `php5-easybib`) without any customizations, etc..

File: php-fpm/recipes/service.rb [hunk 1]
```diff
@@ -1,18 +1,18 @@
...
+               end
 
 service 'php-fpm' do
   action :nothing
-  provider service_provider
+  unless node['php']['ppa']['package_prefix'].include?('easybib')
+    if node['platform_version'] == '16.04'
+      provider Chef::Provider::Service::Systemd
+    else
+      provider Chef::Provider::Service::Upstart
```
Comment (fh) [inline php-fpm/recipes/service.rb]: @till Why do you hardcode a provider and dont let chef autodetect here?

Comment (till) [inline php-fpm/recipes/service.rb]: @fh it was a `16.04` issue during testing. It detected the wrong one.

Comment (till) [inline php-fpm/recipes/service.rb]: That could be a problem with Chef11 on 16.04.

**Comment by fh** [comment]
Tested on a instance, works.

**Pull request merged**